setup.py

```diff
@@ -101,34 +100,35 @@
         "ipython>=8.5.0,<9",
         "lazyasd>=0.1.4",
         "packaging>=23.0,<24",
-        "pandas>=1.3.0,<2",
+        "pandas>=2",
         "pluggy>=1.3,<2",
         "pydantic>=2.5.2,<3",
         "pydantic-settings>=2.0.3,<3",
         "PyGithub>=1.59,<2",
         "pytest>=6.0,<8.0",
-        "python-dateutil>=2.8.2,<3",
+        "python-dateutil>=2.8.2",
```

Comment:
- [ ] put back all upper bounds as you can

Reply:
done by resetting dependencies (moved to another PR in https://github.com/ApeWorX/ape/pull/2003) in f04d70289737d69453a356398544c917cb547be5